jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -301,6 +219,17 @@ With the following, nested under `steps:` and customizing for your application a
 - See the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy) document for valid `deploy` options to configure deployments on CircleCI 2.0
 - Please read the [Deploy]({{ site.baseurl }}/2.0/deployment-integrations/) document for examples of deployment for CircleCI 2.0.
 
+## Environment Variables
+
+In CircleCI 2.0, all defined values are treated literally.
+When defining configuration variables like `working_directory` or `images`,
+you will not be able to interpolate other variables.
+However, you **can** interpolate a variable within a command
+by setting it for the current shell.
+
+For full details,
```

Comment:
'full details' is awkward for translation (full is colloquial here)
also link is pointing to localhost